client.go

```diff
@@ -159,6 +168,18 @@ func DefaultRetryPolicy(resp *http.Response, err error) (bool, error) {
 	return false, nil
 }
 
+// DefaultBackoffPolicy provides a default callback for Client.Backoff which
+// will perform exponential backoff based on the attempt number and limited
+// by the provided minimum and maximum durations.
+func DefaultBackoffPolicy(min, max time.Duration, attemptNum int) time.Duration {
```

Comment:
This should probably be named `DefaultBackoff` as the type is simply named `Backoff`.